service-commands/scripts/seed.js

```diff
@@ -3,10 +3,7 @@ const ServiceCommands = require('../src/index')
 const _ = require('lodash')
 const colors = require('colors')
 
-const {
-    SeedSession,
-    SeedUtils
-} = ServiceCommands
+const { SeedSession, SeedUtils } = ServiceCommands
```

Comment:
what are the changes in this file?

Reply:
These are all just what my linter did on save

Reply:
cool - ideally annotate the diffs if possible to make reviews easier 🙏  i also find it helps me remember what i did lol